core/domain/prod_validation_jobs_one_off.py

```diff
@@ -5130,10 +5130,12 @@ def map(model_instance):
             validator.validate(model_instance)
             if len(validator.errors) > 0:
                 for error_key, error_val in validator.errors.items():
+                    failure_messages = set([
+                        'FAILED: %s' % error for error in error_val])
                     yield (
                         'failed validation check for %s of %s' % (
                             error_key, model_name),
-                        (',').join(set(error_val)))
+                        (',').join(failure_messages))
```

Comment:
I think actually you want the FAILED: at the beginning  of "failed validation check". The idea is to have "FAILED" at the start of the bullet point, not all over the place, because in the latter case, Ctrl+F would be a bit useless since the word would show up everywhere.

Reply:
I do not understand the use case there, why can't `failed validation check` be used directly then for Ctrl + F?

Reply:
It's just for clarity. If "failed validation check" is always the string to grep for that clearly indicates "failed", and if this string appears only once per bullet point in the admin jobs dashboard, then that is fine too -- I can use that.

(If it's not currently consistent, then using "FAILED: " everywhere instead might be a good idea.)

Reply:
Yes, it is consistent. It will appear once per bullet point: https://github.com/oppia/oppia/blob/develop/core/domain/prod_validation_jobs_one_off.py#L5134

Reply:
Ah, ok. Then in that case I think this PR is not needed (sorry for the miscommunication)!

Reply:
No issues!